Covid19Radar/Tests/Covid19Radar.UnitTests/Services/HttpDataServiceTests.cs

```diff
@@ -139,13 +140,19 @@ public async Task PostRegisterUserAsyncTestsAsync_Exception()
 
             var unitUnderTest = CreateService();
 
-            var result = await unitUnderTest.PostRegisterUserAsync();
+            try
+            {
+                var result = await unitUnderTest.PostRegisterUserAsync();
+
```

Comment:
このテストコードだと例外が発生しなくても成功してしまう気がするので、 `unitUnderTest.PostRegisterUserAsync()` の実行直後に `Assert.Fail()` が欲しいです。

```suggestion
                Assert.Fail("Exception should be raised.");
```

Reply:
`Assert.ThrowsAsync` に変えたので `Assert.Fail()` はいらないかなと思います。
ご確認お願いします。